nautobot/core/views/generic.py

```diff
@@ -948,6 +957,10 @@ def post(self, request, **kwargs):
         logger = logging.getLogger("nautobot.views.BulkEditView")
         model = self.queryset.model
 
+        relationships_errors = get_relationships_errors(request, model)
```

Comment:
Is it correct that here we're passing the model class as the second parameter, while in the single-edit case we're passing `obj` (model instance) instead? Does `get_relationships_errors` handle both cases?

Reply:
Yeah, seems to!